config/Common.xcconfig

```diff
@@ -1,4 +1,4 @@
-IPHONEOS_DEPLOYMENT_TARGET = 15.0
+IPHONEOS_DEPLOYMENT_TARGET = 16.0
```

Comment:
Already set in the project but not update here and ignored because some target overrode it instead of inheriting (by deleting the entry at the target level)